### ISSUE TITLE
Error object has `.sqlState` instead of `.code` when using pg-native

### ISSUE THREAD

**Issue by OJezu** [brianc/node-postgres#736]
Error without native:

```
{ [error: relation "id_5_col" already exists]
  name: 'error',
  length: 94,
  severity: 'ERROR',
  code: '42P07',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'index.c',
  line: '773',
  routine: 'index_create' }
```

Same error with native

```
{ [Error: ERROR:  relation "id_5_col" already exists
]
  severity: 'ERROR',
  sqlState: '42P07',
  messagePrimary: 'relation "id_5_col" already exists',
  sourceFile: 'index.c',
  sourceLine: '773',
  sourceFunction: 'index_create' }
```

**Comment by kinetics** [comment]
+1

**Comment by linpy** [comment]
I am out of the office until 2015/07/24.

I would on leave from 2015/1/22~1/25. Please drop me mail if anything.
Thanks.


Note: This is an automated response to your message  "Re: [node-postgres]
Error object has `.sqlState` instead of `.code` when using pg-native
(#736)" sent on 22/07/2015 5:54:31.

This is the only notification you will receive while this person is away.

**Comment by bendrucker** [comment]
@brianc please block @linpy when you have a sec so his/her autoresponder stops spamming issues

**Comment by linpy** [comment]
I am out of the office until 2015/07/24.

I would on leave from 2015/1/22~1/25. Please drop me mail if anything.
Thanks.


Note: This is an automated response to your message  "Re: [node-postgres]
Error object has `.sqlState` instead of `.code` when using pg-native
(#736)" sent on 22/07/2015 11:31:00.

This is the only notification you will receive while this person is away.

**Comment by brianc** [comment]
This will be fixed in `pg@7.0`
